vignettes/markdown.Rmd

```diff
@@ -141,33 +148,52 @@ URLs inside angle brackets are also automatically converted to hyperlinks:
-
-```r
-#' * [terms][terms.object] becomes \link[=terms.object]{terms}
-#' * [abc][abc-class] becomes \link[=abc-class]{abc}
-```
-
-Or a shorthand notation can be used: `[abc-class]` is converted to `\linkS4class{abc}` and `[pkg::abc-class]` is converted to `\link[pkg:abc-class]{pkg::abc}`.
+Markdown notation can also be used to create links to other help topics. There are two basic forms:
+
+  * `[ref]`: The target topic and the link text are the same.
+  * `[text][ref]`: Link text differs from the target.
+
+with variations depending on whether `ref` is in the same package and whether the link is automatically typeset as code. Use backticks inside the square brackets to typeset the link text as code, when it would not be otherwise:
+
+  * ``[`ref`]``
+  * ``[`text`][ref]``
+
++-----------------------+------------------------+--------------------------+
+| Examples              | Help topic\            | Notes                    |
+|                       | for what?              |                          |
++=======================+========================+==========================+
+| `[func()]`            | a function in\         | Always typeset as code.\            |
+|                       | same package           | In Rd: `\code{\link[=func]{func()}` | 
++-----------------------+------------------------+--------------------------+  
+| `[object]`\           | a topic in\            | Use for a topic where you documented\ |
+| ``[`object`]``        | same package           | `NULL` and set its name via `@name`,\ |
+|                       |                        | e.g., a class, dataset, or theme.\    | 
+|                       |                        | Bring your own backticks.\            |
+|                       |                        | In Rd: `\link{object}`                |
++-----------------------+------------------------+--------------------------+
+| `[text][func()]`\     | a function or topic\   | Use when you want to provide your\               |
+| `[text][object]`\     | in same package        | own link text.\                                  |
+| `[abc][abc-class]`    |                        | Bring your own backticks.\                       |
+|                       |                        | In Rd: `\link[=func]{text}`,                     |
+|                       |                        | `\link[=obj]{text}`, or `\link[=abc-class]{abc}` |
++-----------------------+------------------------+--------------------------+
+| `[pkg::func()]`       | a function in `pkg`    | Always typeset as code.\                     |
+|                       |                        | In Rd: `\code{\link[pkg:func]{pkg::func()}}` |
++-----------------------+------------------------+--------------------------+
+| `[pkg::object]`       | a topic in `pkg`       | Use for a non-function object in `pkg`\ |
+|                       |                        | Bring your own backticks.\              |
```

Comment:
I am not even sure if you can use backticks in this case. Can you?

Reply:
It appears to work, yes. I just tried.